flashlight/fl/examples/RnnClassification.cpp

```diff
@@ -151,21 +151,49 @@ class RnnClassifier : public Container {
       unsigned vocabSize,
       int hiddenSize = 256,
       unsigned numLayers = 2)
-      : embed_(Embedding(hiddenSize, vocabSize)),
-        rnn_(
-            RNN(hiddenSize,
-                hiddenSize,
-                numLayers,
-                RnnMode::GRU,
-                false /* Dropout */)),
-        linear_(Linear(hiddenSize, numClasses)),
+      : embed_(std::make_shared<Embedding>(hiddenSize, vocabSize)),
+        rnn_(std::make_shared<RNN>(
+            hiddenSize,
+            hiddenSize,
+            numLayers,
+            RnnMode::GRU,
+            0 /* Dropout */)),
+        linear_(std::make_shared<Linear>(hiddenSize, numClasses)),
         logsoftmax_(0) {
     std::cout << "Creating a RNN Classifier with vocab size: " << vocabSize
               << " and num classes: " << numClasses << std::endl;
+    createLayers();
+  }
+
+  RnnClassifier(const RnnClassifier& other) {
+    copy(other);
+    createLayers();
+  }
+
+  RnnClassifier(RnnClassifier&& other) = default;
+
+  RnnClassifier& operator=(const RnnClassifier& other) {
+    clear();
+    copy(other);
+    createLayers();
```

Comment:
copy assignment operator missing a return value

Reply:
Oops. I guess the examples aren't compiled in the CI now (if they ever were)?